bin/cron_admin_stats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env ruby
+#
+# This script caches database queries whose results are displayed at studio.code.org/admin/stats.
+#
+require File.expand_path('../../pegasus/src/env', __FILE__)
+require src_dir 'database'
+require 'cdo/hip_chat'
+require 'cdo/properties'
+
+DASHBOARD_REPORTING_DB_READONLY = sequel_connect(CDO.dashboard_reporting_db_reader, CDO.dashboard_reporting_db_reader)
+PEGASUS_REPORTING_DB_READONLY = sequel_connect(CDO.pegasus_reporting_db_reader, CDO.pegasus_reporting_db_reader)
+
+def main()
+  return unless only_one_running?(__FILE__)
+
+  users = DASHBOARD_REPORTING_DB_READONLY[:users]
+  students = users.where(:user_type => 'student')
+  teachers = users.where(:user_type => 'teacher')
+  followers = DASHBOARD_REPORTING_DB_READONLY[:followers]
+  prizes = DASHBOARD_REPORTING_DB_READONLY[:prizes]
+  teacher_prizes = DASHBOARD_REPORTING_DB_READONLY[:teacher_prizes]
+  teacher_bonus_prizes = DASHBOARD_REPORTING_DB_READONLY[:teacher_bonus_prizes]
+
+  Properties.set(:admin_stats, {
+    user_count: users.count,
+    teacher_count: teachers.count,
+    student_count: students.count,
+    users_with_teachers: followers.distinct.count(:student_user_id),
+    users_with_email: users.where('email <> ""').count,
+    users_with_confirmed_email: users.where('confirmed_at IS NOT NULL').count,
+    girls: students.where(:gender => 'f').count,
+    boys: students.where(:gender => 'm').count,
+
+    prizes_redeemed: prizes.where('user_id IS NOT NULL').group(:prize_provider_id).count,
```

Comment:
I don't think these prizes things are used. (these are the prizes from 2013 hoc, the 2015 hoc prizes are handled elsewhere)